AppCenter/AppCenterTests/MSAppCenterTests.m

```diff
@@ -46,22 +50,32 @@ @implementation MSAppCenterTest
 - (void)setUp {
   [super setUp];
   [MSAppCenter resetSharedInstance];
+  [MSAuthTokenContext resetSharedInstance];
   [MSUserIdContext resetSharedInstance];
 
   // System Under Test.
   self.sut = [[MSAppCenter alloc] init];
-
   self.settingsMock = [MSMockUserDefaults new];
+  self.channelGroupMock = OCMClassMock([MSChannelGroupDefault class]);
+  self.channelUnitMock = OCMProtocolMock(@protocol(MSChannelUnitProtocol));
+  OCMStub([self.channelGroupMock alloc]).andReturn(self.channelGroupMock);
+  OCMStub([self.channelGroupMock initWithInstallId:OCMOCK_ANY logUrl:OCMOCK_ANY]).andReturn(self.channelGroupMock);
+  OCMStub([self.channelGroupMock addChannelUnitWithConfiguration:OCMOCK_ANY]).andReturn(self.channelUnitMock);
+
+  // Device tracker.
   [MSDeviceTracker resetSharedInstance];
   self.deviceTrackerMock = OCMClassMock([MSDeviceTracker class]);
   OCMStub([self.deviceTrackerMock sharedInstance]).andReturn(self.deviceTrackerMock);
+
+  // Session context.
   [MSSessionContext resetSharedInstance];
   self.sessionContextMock = OCMClassMock([MSSessionContext class]);
   OCMStub([self.sessionContextMock sharedInstance]).andReturn(self.sessionContextMock);
 }
 
 - (void)tearDown {
   [self.settingsMock stopMocking];
+  [self.channelGroupMock stopMocking];
```

Comment:
```suggestion
  [self.channelGroupMock stopMocking];
  [self.channelUnitMock stopMocking];
```

Reply:
No, we shouldn't do it for `OCMProtocolMock`